rats-tls/dist/rpm/rats_tls.spec.in

```diff
@@ -51,27 +75,63 @@ if [ ! -d "$SGX_SDK" ]; then
+make -C build-occlum DESTDIR=%{?buildroot}/build-occlum install
+cmake -DRATS_TLS_BUILD_MODE="sgx"  -DBUILD_SAMPLES=on -H. -Bbuild-sgx
+make -C build-sgx DESTDIR=%{?buildroot}/build-sgx install
+cmake -DRATS_TLS_BUILD_MODE="tdx"  -DBUILD_SAMPLES=on -H. -Bbuild-tdx
+make -C build-tdx DESTDIR=%{?buildroot}/build-tdx install
 popd
 
 %postun
 rm -rf %{RATS_TLS_LIBDIR} %{RATS_TLS_INCDIR} $(dirname %{RATS_TLS_BINDIR})
 
-%files
-%{RATS_TLS_BINDIR}/rats-tls-server
-%{RATS_TLS_BINDIR}/rats-tls-client
-%{RATS_TLS_INCDIR}/rats-tls/*.h
-%{RATS_TLS_LIBDIR}/librats_tls.so*
-%{RATS_TLS_LIBDIR}/tls-wrappers/libtls_wrapper*.so*
-%{RATS_TLS_LIBDIR}/crypto-wrappers/libcrypto_wrapper*.so*
-%{RATS_TLS_LIBDIR}/attesters/libattester*.so*
-%{RATS_TLS_LIBDIR}/verifiers/libverifier*.so*
+%files host
+/build-host/usr/share/rats-tls/samples/rats-tls-server
+/build-host/usr/share/rats-tls/samples/rats-tls-client
+/build-host/usr/local/include/rats-tls/rats-tls/*.h
+/build-host/usr/local/lib/rats-tls/librats_tls.so*
+/build-host/usr/local/lib/rats-tls/tls-wrappers/libtls_wrapper*.so*
+/build-host/usr/local/lib/rats-tls/crypto-wrappers/libcrypto_wrapper*.so*
+/build-host/usr/local/lib/rats-tls/attesters/libattester*.so*
+/build-host/usr/local/lib/rats-tls/verifiers/libverifier*.so*
+
+%files occlum
+/build-occlum/usr/share/rats-tls/samples/rats-tls-server
+/build-occlum/usr/share/rats-tls/samples/rats-tls-client
+/build-occlum/usr/local/include/rats-tls/rats-tls/*.h
+/build-occlum/usr/local/lib/rats-tls/librats_tls.so*
+/build-occlum/usr/local/lib/rats-tls/tls-wrappers/libtls_wrapper*.so*
+/build-occlum/usr/local/lib/rats-tls/crypto-wrappers/libcrypto_wrapper*.so*
+/build-occlum/usr/local/lib/rats-tls/attesters/libattester*.so*
+/build-occlum/usr/local/lib/rats-tls/verifiers/libverifier*.so*
+
+%files sgx
+/build-sgx/usr/share/rats-tls/samples/rats-tls-server
+/build-sgx/usr/share/rats-tls/samples/rats-tls-client
+/build-sgx/usr/share/rats-tls/samples/sgx_stub_enclave.signed.so
+/build-sgx/usr/local/include/rats-tls/rats-tls/*.h
+/build-sgx/usr/local/lib/rats-tls/librats_tls.a
+/build-sgx/usr/local/lib/rats-tls/librtls_edl_t.a
+/build-sgx/usr/local/lib/rats-tls/librats_tls_u.a
+/build-sgx/usr/local/lib/rats-tls/tls-wrappers/libtls_wrapper*.a
+/build-sgx/usr/local/lib/rats-tls/crypto-wrappers/libcrypto_wrapper*.a
+/build-sgx/usr/local/lib/rats-tls/attesters/libattester*.a
+/build-sgx/usr/local/lib/rats-tls/verifiers/libverifier*.a
+
+%files tdx
+/build-tdx/usr/share/rats-tls/samples/rats-tls-server
+/build-tdx/usr/share/rats-tls/samples/rats-tls-client
+/build-tdx/usr/local/include/rats-tls/rats-tls/*.h
+/build-tdx/usr/local/lib/rats-tls/librats_tls.so*
+/build-tdx/usr/local/lib/rats-tls/tls-wrappers/libtls_wrapper*.so*
+/build-tdx/usr/local/lib/rats-tls/crypto-wrappers/libcrypto_wrapper*.so*
+/build-tdx/usr/local/lib/rats-tls/attesters/libattester*.so*
+/build-tdx/usr/local/lib/rats-tls/verifiers/libverifier*.so*
 
```

Comment:
Please put the following content to a file so that you can use `%files host -f %{SOURCE10}` to reduce the line 94-101,  %files occlum -f %{SOURCE10}` to reduce the line 104-111, 
```
build-tdx/usr/share/rats-tls/samples/rats-tls-server
/build-tdx/usr/share/rats-tls/samples/rats-tls-client
/build-tdx/usr/local/include/rats-tls/rats-tls/*.h
/build-tdx/usr/local/lib/rats-tls/librats_tls.so*
/build-tdx/usr/local/lib/rats-tls/tls-wrappers/libtls_wrapper*.so*
/build-tdx/usr/local/lib/rats-tls/crypto-wrappers/libcrypto_wrapper*.so*
/build-tdx/usr/local/lib/rats-tls/attesters/libattester*.so*
/build-tdx/usr/local/lib/rats-tls/verifiers/libverifier*.so*
```

Reply:
I don't think that format using "-f {Source}" is a better style.  And current style will be more simple in file directory.

Reply:
To keep directory clean,that's why i giving a workaround to change Makefile in last verison PR.